inc/admin/options-page.php

```diff
@@ -144,7 +156,19 @@ public function register_settings_and_fields() {
 			'social_media_links',
 			array(
 				'id'          => 'youtube_url',
-				'description' => __( 'Enter your Youtube channel URL', 'alcatraz' ),
+				'description' => __( 'Enter your YouTube channel URL', 'alcatraz' ),
+			)
+		);
+
+		add_settings_field(
+			'show_social_urls',
```

Comment:
I'd change this option key to 'social_icons_in_footer'. To me 'show_social_urls' doesn't indicate where they are shown, or what is actually shown.